docs/quick-start.md

```diff
@@ -214,6 +286,24 @@ tens or hundreds of nodes. You can also do this interactively by connecting `bin
 a cluster, as described in the [programming guide](programming-guide.html#initializing-spark).
 
 </div>
+<div data-lang="r" markdown="1">
+
+{% highlight r %}
+> cache(linesWithSpark)
+
+> count(linesWithSpark)
```

Comment:
This number may be outdated (The readme could be updated without touching these examples).